Data/data.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I am confused as to why you need a json here? Would it not be easier to have the data in the same place?